health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -529,12 +456,13 @@ async function getUserWithMobileNumbers(
   }
   // Convert the results array to a Set to eliminate duplicates
   const resultSet = new Set(allResults);
-  logger.info(`Already Existing mobile numbers : ${JSON.stringify(resultSet)}`);
+  logger.info(`Already Existing mobile numbers : ${Array.from(resultSet).join(",")}`);
   return resultSet;
 }
 
 async function matchUserValidation(createdData: any[], request: any) {
-  var count = 0;
+  var mobileNumbercount = 0;
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)